src/screens/Onboarding/StepProfile/SelectImageButton.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import {Pressable, StyleSheet} from 'react-native'
+import {Camera_Stroke2_Corner0_Rounded as Camera} from '#/components/icons/Camera'
+import {useSetAvatar} from '#/screens/Onboarding/StepProfile/index'
+import {useTheme} from '#/alf'
+import {usePhotoLibraryPermission} from 'lib/hooks/usePermissions'
+import {openPicker} from 'lib/media/picker.shared'
+import {openCropper} from 'lib/media/picker'
+import {compressIfNeeded} from 'lib/media/manip'
+import {isWeb} from 'platform/detection'
+
+export function SelectImageButton() {
+  const t = useTheme()
+  const setAvatar = useSetAvatar()
+  const {requestPhotoAccessIfNeeded} = usePhotoLibraryPermission()
+
+  const onCameraPress = React.useCallback(async () => {
+    if (!(await requestPhotoAccessIfNeeded())) {
+      return
+    }
+
+    const items = await openPicker({
+      aspect: [1, 1],
+    })
+    const item = items[0]
+    if (!item) return
+
+    // TODO we need an alf modal for the cropper
+    if (isWeb) {
+      const compressedImage = await compressIfNeeded(item)
+
+      setAvatar(prev => ({
+        ...prev,
+        image: compressedImage,
+      }))
+      return
+    }
```

Comment:
A hack fix for now because we don't have the web cropper modal working with ALF, and we can't see the legacy modals while in the onboarding flow.